packages/tools/devtools/devtools/README.md

```diff
@@ -26,10 +26,60 @@ The Devtools' API surface is designed to fit nicely into most application flows.
 ### Initialization
 
 To initialize a devtools session for your container, call `initializeDevtools`.
-This function accepts a `DevtoolsLogger` for recording and communicating telemetry data, a list of initial Fluid `Containers` to associate with the session, and (optionally) customized data visualization configurations for visualizing `Container` data.
+This function accepts a `DevtoolsLogger` for receiving Fluid telemetry from your application, a list of initial Fluid
+`Containers` to associate with the session, and (optionally) customized data visualization configurations for visualizing
+`Container` data.
 
 TODO: link to API docs once API shape has settled.
```

Comment:
Can we link to the docs here? https://fluidframework.com/docs/apis/devtools#initializedevtools-function